start.py

```diff
@@ -490,10 +506,13 @@ def failure(timeout):
                        timeout=180,
                        success=success, failure=failure)
 
+    if kerberos_ticket_lifetime:
+        cm_kerberos_principals = deployment.get_cm_kerberos_principals()
+        logger.info('cm_kerberos_principals = %s', cm_kerberos_principals)
+        _apply_kerberos_ticket_expiration_for_cm_principals(cluster, cm_kerberos_principals, kerberos_ticket_lifetime)
     for service in deployment.get_cluster_services(cluster_name=DEFAULT_CLUSTER_NAME):
         if service['type'] == 'HUE':
             _apply_kerberos_fix_for_hue(cluster)
-            break
```

Comment:
Why was this removed as part of the change?

Reply:
Done.